docs/pages/management/dynamic-resources.mdx

```diff
@@ -59,6 +59,10 @@ to manipulate cluster resources stored on the Auth Service backend. The design
 of Teleport's configuration interface makes it well suited for
 infrastructure-as-code and GitOps approaches.
 
+You can get started with `tctl`, the Terraform Provider or the Kubernetes
```

Comment:
```suggestion
You can get started with `tctl`, the Terraform Provider, and the Kubernetes
```
- Using the Oxford comma for consistency with the rest of the guide. Changing the conjunction to "and".